iree/BUILD.bazel

```diff
@@ -44,3 +44,11 @@ config_setting(
         "iree_is_msvc": "true",
     },
 )
+
+config_setting(
+    name = "macos_x86_64",
```

Comment:
Would be nice if there was a shared config for this in `@bazel_tools`. Otherwise, putting this here seems fine to me.

Reply:
`"@bazel_tools//src/conditions:darwin"`?

Reply:
There's one specifically for x86_64 as well, `darwin_x86_64`:

https://github.com/bazelbuild/bazel/blob/dc9e15d0991392fe54769ae48f6af44661b83f00/src/conditions/BUILD#L82-L89

Reply:
Yes, will change to `@bazel_tools//src/....`